src/demo-app/input/input-container-demo.html

```diff
@@ -260,11 +260,11 @@
 <md-card class="demo-card demo-basic">
   <md-toolbar color="primary">Textarea Autosize</md-toolbar>
   <md-card-content>
-    <textarea md-autosize class="demo-textarea"></textarea>
+    <textarea md-textarea-autosize class="demo-textarea"></textarea>
```

Comment:
Updated it. Didn't add it for `matTextareaAutosize` because according to PR #2244 it's not needed.